docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: "Creating custom Declarative Automation conditions | Dagster Docs"
+description: "Learn to create your own custom Declarative Automation conditions."
+---
+
+# Creating custom Declarative Automation conditions
+
+<Note>
+  Declarative Automation is currently <strong>experimental</strong>.
+</Note>
+
+[Declarative Automation](/concepts/automation/declarative-automation) includes pre-built conditions to handle common use cases, such as executing on a periodic schedule or whenever an upstream dependency updates, but you can also customize conditions.
+
+By the end of this guide, you'll understand how <PyObject object="AutomationCondition" pluralize /> work and how to create your own custom conditions.
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Asset definitions](/concepts/assets/software-defined-assets)
+- [Declarative Automation](/concepts/automation/declarative-automation)
+
+---
+
+## How it works
+
+Each <PyObject object="AutomationCondition" /> consists of a set of **operands** and various **operators**. To create conditions that suit your specific needs, you can combine the operators and operands listed below with Dagster's built-in <PyObject object="AutomationCondition" pluralize />. For example:
```

Comment:
nit: I would consider all of the operators and operands to essentially be "built-in AutomationConditions", so you're really just combining them together in general, maybe ust:

"To create conditions that suit your specific needs, you can combine the operators and operands listed below."